### DISCUSSION TITLE
Welcome to PsychoStats Discussions!

### DISCUSSION THREAD

**Discussion by Drek282** [Drek282/PsychoStats#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by angelperezleon** [comment]
As always great work on keeping this project alive for us all to continue enjoying using it, works fantastically.
Your new ideas I can wait to see them implemented, they will be top-gun stuff!

remember amxbans? maybe integration one day into this for banning purposes on servers...
